migration/migration_blackbox_test.go

```diff
@@ -299,6 +301,56 @@ func testMigration25ValidMiss(t *testing.T) {
 
 }
 
+func testMigration27(t *testing.T) {
+	migrateToVersion(sqlDB, migrations[:(28)], (28))
+
+	// Confirm that the manage_members scope was added
+	rows, err := sqlDB.Query("SELECT resource_type_scope_id FROM resource_type_scope rts, resource_type rt WHERE rts.resource_type_id = rt.resource_type_id AND rt.name = 'identity/organization' AND rts.name = 'manage_members'")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer rows.Close()
+
+	require.True(t, rows.Next())
+	var resourceTypeScopeID uuid.UUID
+	err = rows.Scan(&resourceTypeScopeID)
+
+	// Now confirm that the scope has been assigned to the organization 'owner' role
+	rows, err = sqlDB.Query("SELECT r.name FROM role r, role_scope rs WHERE r.role_id = rs.role_id AND rs.scope_id = $1", resourceTypeScopeID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer rows.Close()
+
+	require.True(t, rows.Next())
+	var roleName string
+	err = rows.Scan(&roleName)
+
+	require.Equal(t, "owner", roleName)
+
+	// Create some test data
+	require.Nil(t, runSQLscript(sqlDB, "026-insert-test-invitation-data.sql"))
+
+	// Confirm that we can create an invitation for an organization
+	_, err = sqlDB.Exec("INSERT INTO invitation (invitation_id, invite_to, identity_id, member) VALUES (uuid_generate_v4(), 'c62d77b2-194c-47d0-8bbf-b1308576876d', 'd9161547-5263-4c83-a729-e39ff088978e', true)")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Confirm that we can create an invitation for a resource
+	_, err = sqlDB.Exec("INSERT INTO invitation (invitation_id, resource_id, identity_id, member) VALUES (uuid_generate_v4(), 'c6a2ee2e-7ec6-4c04-ae7e-5ff8c36b28b9', 'd9161547-5263-4c83-a729-e39ff088978e', false)")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Confirm that we get a check constraint violation if we try to provide both invite_to and resource_id values
+	_, err = sqlDB.Exec("INSERT INTO invitation (invitation_id, invite_to, resource_id, identity_id, member) VALUES (uuid_generate_v4(), 'c62d77b2-194c-47d0-8bbf-b1308576876d', 'c6a2ee2e-7ec6-4c04-ae7e-5ff8c36b28b9', 'd9161547-5263-4c83-a729-e39ff088978e', false)")
+	require.NotNil(t, err)
+
+	// Cleanup the test data
+	require.Nil(t, runSQLscript(sqlDB, "026-cleanup-test-invitation-data.sql"))
```

Comment:
optional, but good practice - since we anyway drop this db where we test migrations. ( after this test suite is done executing )